### PR TITLE
Magic

### PR DESCRIPTION
**Pull request by ade1705**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+            <h1>Howdy, <br>You have eleven items yet to be completed in your toDo.</h1>
+            <div class="padding-20 shadow rounded" data-binding="todoList">
+
+            </div>
+        </div>
+        <div class="flex-1 img-bg"></div>
+    </div>
+</body>
+<style>
+    @import url('https://fonts.googleapis.com/css?family=Poppins:400,500,600,700&display=swap');
```
Comment (oluwajuwon) [inline index.html]: WHAT HAPPENED TO A REGULAR CSS FILE NA?

File: src/TodoHtmlCreator.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
+export default class TodoHtmlCreator
```
Comment (oluwajuwon) [inline src/TodoHtmlCreator.js]: OH MAD MAD

File: src/ToDoApp.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import {ToDoClickHandler} from "./ToDoClickHandler";
+
+
+export default class ToDoApp {
```
Comment (oluwajuwon) [inline src/ToDoApp.js]: GUY THIS IS FUCKING MAD OOOOO

File: test/ToDoRepository.test.js [hunk 4]
```diff
@@ -1,9 +1,9 @@
-const toDo = require('../ToDo');
-const toDoRepository = require('../ToDoRepository');
+import ToDoRepository from "../src/ToDoRepository";
+import ToDo from "../src/ToDo";
```
Comment (estherkoko) [inline test/ToDoRepository.test.js]: this is cool but on a wider team, relative path is probably not ideal

**Pull request merged**